python/test/CMakeLists.txt

```diff
@@ -261,6 +261,7 @@ ot_pyinstallcheck_test (LevelSet_std)
 ot_pyinstallcheck_test (LevelSetMesher_std)
 ot_pyinstallcheck_test (Mesh_std)
 ot_pyinstallcheck_test (Mesh_draw IGNOREOUT)
+ot_pyinstallcheck_test (MeshBoundaryFactory_std IGNOREOUT)
```

Comment:
you needs some asserts and remove the .expout

Reply:
done